src/app/file/pixly_format.cpp

```diff
@@ -0,0 +1,535 @@
+// Aseprite
+// Copyright (C) 2016  Carlo "zED" Caputo
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 2 as
+// published by the Free Software Foundation.
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "app/app.h"
+#include "app/document.h"
+#include "app/file/file.h"
+#include "app/file/file_format.h"
+#include "app/file/format_options.h"
+#include "app/ini_file.h"
+#include "app/xml_document.h"
+#include "base/file_handle.h"
+#include "base/convert_to.h"
+#include "base/path.h"
+#include "doc/doc.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <cctype>
+
+#include "png.h"
+
+namespace app {
+
+using namespace base;
+
+class PixlyFormat : public FileFormat {
+  const char* onGetName() const override { return "pixly"; }
+  const char* onGetExtensions() const override { return "anim"; }
+  int onGetFlags() const override {
+    return
+      FILE_SUPPORT_LOAD |
+      FILE_SUPPORT_SAVE |
+      FILE_SUPPORT_RGB |
+      FILE_SUPPORT_RGBA |
+      FILE_SUPPORT_GRAY |
+      FILE_SUPPORT_GRAYA |
+      FILE_SUPPORT_INDEXED |
+      FILE_SUPPORT_LAYERS |
+      FILE_SUPPORT_FRAMES |
+      FILE_SUPPORT_PALETTE_WITH_ALPHA;
```

Comment:
Remove `FILE_SUPPORT_PALETTE_WITH_ALPHA` too as this format doesn't support indexed images with color palettes (with alpha or without alpha).